engine/suspender.go

```diff
@@ -128,16 +142,19 @@ func (eng *Engine) Suspender(ctx context.Context, cs *kubernetes.Clientset) {
 			- now, which contains today's time
 			- suspendAt, which contains today's suspend time for the namespace
 		*/
-
+		stepName = "Handle deiredState"
```

Comment:
`stepName` string does not begin with an uppercase letter in the two others definitions